src/cogs/esports/views/scrims/_edit.py

```diff
@@ -62,7 +62,7 @@ async def initial_message(self):
             "Success Role": getattr(scrim.role, "mention", "`role-deleted`"),
             "Mentions": f"`{scrim.required_mentions}`",
             "Slots": f"`{scrim.total_slots}`",
```

Comment:
this is correct but we actually need the old format too , I mentioned that in the issue as well, that the old format must be included in the brackets after the new format. 

What I mean is:
`new_format (old_format)`